packages/rocketchat-i18n/i18n/en.i18n.json

```
@@ -3548,6 +3550,7 @@
   "SlackBridge_Out_Channels_Description": "Choose which channels will send messages back to Slack",
   "SlackBridge_Out_Enabled": "SlackBridge Out Enabled",
   "SlackBridge_Out_Enabled_Description": "Choose whether SlackBridge should also send your messages back to Slack",
+  "SlackBridge_Remove_Channel_Links_description": "Remove the internal link between Rocket.Chat channels and Slack channels. The links will afterwards be recreated based on the channel names.",
```

Took me a while to find out why the description was not showing on Settings screen
```suggestion
  "SlackBridge_Remove_Channel_Links_Description": "Remove the internal link between Rocket.Chat channels and Slack channels. The links will afterwards be recreated based on the channel names.",
```